presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -213,7 +214,12 @@ public static PrestoSparkQueryRunner createHivePrestoSparkQueryRunner(Iterable<T
 
     public static PrestoSparkQueryRunner createHivePrestoSparkQueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> additionalConfigProperties, Optional<Path> dataDirectory)
     {
-        PrestoSparkQueryRunner queryRunner = new PrestoSparkQueryRunner("hive", additionalConfigProperties, dataDirectory);
+        return createHivePrestoSparkQueryRunner(tables, additionalConfigProperties, dataDirectory, NODE_COUNT);
```

Comment:
Where is NODE_COUNT defined?

Reply:
Static field in the same class. Changed all the names now to make it more clear whats happening